src/test/java/uk/nhs/prm/deduction/e2e/queue/QueueMessageHelper.java

```diff
@@ -97,6 +121,18 @@ private SqsMessage findMessageContaining(String substring) {
         return null;
     }
 
+    private ArrayList<SqsMessage> findAllMessagesContaining(String substring) {
```

Comment:
```suggestion
    private List<SqsMessage> findAllMessagesContaining(String substring) {
```

Can we change this to the generic List interface please?